cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/JarResolverImpl.java

```diff
@@ -73,24 +73,28 @@
     private final String balHomePath;
     private final Manifest manifest;
     private boolean skipCopyLibsFromDist;
+    private boolean isChoreoExtensionSkipped;
```

Comment:
Lets rename to all instances of Choreo to observability since this is a generic feature